utils/remote.cc

```diff
@@ -2397,7 +2397,7 @@ int flush(char const* rpcurl, tr_variant* benc, RemoteConfig& config)
     auto status = EXIT_SUCCESS;
     if (auto const res = curl_easy_perform(curl); res != CURLE_OK)
     {
-        tr_logAddWarn(fmt::format(" ({}) {}", rpcurl_http, curl_easy_strerror(res)));
+        fmt::print(stderr, "{}: {}\n", curl_easy_strerror(res), rpcurl_http);
```

Comment:
Not sure what `curl_easy_strerror()` returns, but I think general approach is to prepend the failure with some description of our own, so maybe
```suggestion
        fmt::print(stderr, "Unable to send request to '{}': {}\n", rpcurl_http, curl_easy_strerror(res));
```

Reply:
Ah, yeah, good call. Applied.